virt/managing_vms/virt-exporting-vms.adoc

```diff
@@ -10,7 +10,7 @@ You can export a virtual machine (VM) and its associated disks in order to impor
 
 You create a `VirtualMachineExport` custom resource (CR) by using the command-line interface.
 
-Alternatively, you can use the xref:../../virt/getting_started/virt-using-the-cli-tools.adoc#vm-volume-export-commands_virt-using-the-cli-tools[`virtctl vmexport` command] to create a `VirtualMachineExport` CR and to download exported volumes.
+Alternatively, you can use the xref:../../virt/getting_started/modules/virt-vm-export-commands.adoc#vm-export-commands_virt-using-the-cli-tools[`virtctl vmexport` command] to create a `VirtualMachineExport` CR and to download exported volumes.
```

Comment:
@jherrman Aha! This is probably what is causing the build failure. I think this is the correct path:

```suggestion
Alternatively, you can use the xref:../../virt/getting_started/virt-using-the-cli-tools.adoc#vm-export-commands_virt-using-the-cli-tools[`virtctl vmexport` command] to create a `VirtualMachineExport` CR and to download exported volumes.
```

Reply:
Interesting, the xref [seems to work fine in the preview](https://97530--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/managing_vms/virt-exporting-vms), but it's still worth a shot.